Changelog.md

```diff
@@ -1,5 +1,36 @@
 # Change log
 
+## 4.0.0 (TBD - not yet released)
+
+> ⚠️ **Deprecation Notice** ⚠️
+>
+> Please note that as of version 4.0.0, the React Apollo project has been deprecated. `@apollo/react-hooks` and `@apollo/react-testing` functionality is now available directly from the `@apollo/client` package (refer to the [Apollo Client 3 migration guide](https://www.apollographql.com/docs/react/) for more details). Both packages still work as is, but their functionality is now re-exported from `@apollo/client`. If you're only using hooks, we recommend using the hooks from `@apollo/client` directly, and dropping your React Apollo dependency(ies). React Apollo's `graphql` HOC and render proper components are now in maintenance mode, meaning they will continue to receive important bug fixes, but will not be updated with new functionality.
```

Comment:
This first sentence is kind of alarming, because it makes it sound like we're removing support for using React with Apollo Client.

Can we frame this more like "You should prefer to import React-related utilities from `@apollo/client`. While we still have separate `@apollo/react-*` packages for ease of migration, they are no longer under active development, as they simply re-export from `@apollo/client`."?

Reply:
Much better - thanks!